### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by 36degrees**
Update a number of dependencies off the back of running `npm audit`. See individual commits for details.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "metalsmith-sass": "^1.5.1",
     "metalsmith-tagcleaner": "0.0.2",
     "metalsmith-uglify": "^2.3.0",
-    "nunjucks": "^3.1.2",
+    "nunjucks": "^3.1.6",
```
Comment (nickcolley) [inline package.json]: I see a bunch of commits but only Nunjucks changed?

Does this mean we could accidentally regress those others given they're only updated in the lock file?

Comment (36degrees) [reply package.json]: I don't think so – they're all dependencies of dependencies?

Comment (nickcolley) [reply package.json]: Is it possible to run `npm audit` as part of CI?

If not, ignore my comments.

Comment (36degrees) [reply package.json]: It's possible, but I'd rather not make it part of this PR if that's OK. I think it's something we should look at in the new year.

**Pull request merged**